### ISSUE TITLE
Invalid URL Error on running development server

### ISSUE THREAD

**Issue by hasanerken** [delvedor/fastify-example#6]
First of all, Thanks for this comprehensive best practice work. 
However, when I run  "npm run dev" I got the error below:

> fastify-app-example@1.0.0 dev
> npm run build && fastify start -w -l info -P app.js

> fastify-app-example@1.0.0 build
> rollup -c

ui/main.js → public/build...
created public/build in 1.3s
TypeError [ERR_INVALID_URL]: Invalid URL: https://undefined.��hq�a��
    at onParseError (internal/url.js:258:9)
    at new URL (internal/url.js:334:5)
    at getUsernameAndPassword

.....

 {
  input: 'https://undefined.��hq�a��',
  code: 'ERR_INVALID_URL'
}

**Comment by delvedor** [comment]
Hello! Can you provide the steps for reproducing the issue?
I've freshly cloned the repository and I'm not encountering the error.

**Comment by chrismilleruk** [comment]
I have been able to reproduce this while following the basic setup steps in the `README`.

This happens if you run elasticsearch locally and place the `address` value from `scripts/prepare-elasticsearch.js` to `ELASTIC_CLOUD_ID` in `.env`.

```dosini
ELASTIC_CLOUD_ID=http://localhost:9200
ELASTIC_API_KEY=...
```

Notably, using `ELASTIC_URL` per the README.md doesn't work. When I change this to `ELASTIC_ADDRESS` it fixes the issue for me.

```dosini
ELASTIC_CLOUD_ID=
ELASTIC_ADDRESS=http://localhost:9200
ELASTIC_API_KEY=...
```

**Comment by delvedor** [comment]
Closed by #8.
